src/devices/Ft232H/Ft232HI2c.cs

```diff
@@ -47,8 +47,15 @@ public override void WriteRead(ReadOnlySpan<byte> writeBuffer, Span<byte> readBu
         /// <inheritdoc/>
         protected override void Dispose(bool disposing)
         {
-            _i2cBus?.RemoveDevice(_deviceAddress);
-            _i2cBus = null!;
+            try
+            {
+                _i2cBus.RemoveDevice(_deviceAddress);
+            }
+            catch (ArgumentException)
```

Comment:
why is RemoveDevice throwing though?

Reply:
It will throw if Dispose is called a second time, which is allowed. Dispose should be idempotent. The previous code won't call RemoveDevice in a second (or subsequent) invocation of Dispose, because _i2cBus will be null.

I confess I do not understand why setting _i2cBus to null in the Dispose method of one instance will cause that field to be null in a new instance.

Reply:
for Device use this pattern: https://github.com/dotnet/iot/blob/main/src/System.Device.Gpio/System/Device/I2c/Devices/UnixI2cDevice.cs#L36-L53 but assume `_shouldDisposeBus` is always false